### PR TITLE
Add checklists for regular and patch releases

### PR DESCRIPTION
**Pull request by schlessera**
This overrides the issue templates provided through the centralized `wp-cli/.github` repository to add two new ones specific to the `wp-cli/wp-cli` repo only.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/1-BUG_REPORT.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+If you are not in the correct repository right now, you can just close this issue/window without submitting and click through to the correct one.
+
+**Are you unsure about what repository to post the bug report into?**
+
+Just head over to the [`wp-cli/wp-cli`](https://github.com/wp-cli/wp-cli) repository and [create a new issue in that repository](https://github.com/wp-cli/wp-cli/issues/new). The maintainers can still move the bug report into the correct repository later on.
+
+--- ✅ If you are in the correct location now... --->
+
+- [ ] Yes, I reviewed the [contribution guidelines](https://make.wordpress.org/cli/handbook/contributing/).
+- [ ] Yes, more specifically, I reviewed the guidelines on [how to write clear bg reports](https://make.wordpress.org/cli/handbook/bug-reports/).
```
Comment (danielbachhuber) [inline .github/ISSUE_TEMPLATE/1-BUG_REPORT.md]:
```suggestion
- [ ] Yes, more specifically, I reviewed the guidelines on [how to write clear bug reports](https://make.wordpress.org/cli/handbook/bug-reports/).
```

File: .github/ISSUE_TEMPLATE/1-BUG_REPORT.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+* `wp scaffold *` https://github.com/wp-cli/scaffold-command
+* `wp search-replace` https://github.com/wp-cli/search-replace-command
+* `wp server` https://github.com/wp-cli/server-command
+* `wp shell` https://github.com/wp-cli/shell-command
+* `wp super-admin *` https://github.com/wp-cli/super-admin-command
+* `wp (widget|sidebar) *` https://github.com/wp-cli/widget-command
+
+If you are not in the correct repository right now, you can just close this issue/window without submitting and click through to the correct one.
+
+**Are you unsure about what repository to post the bug report into?**
```
Comment (danielbachhuber) [inline .github/ISSUE_TEMPLATE/1-BUG_REPORT.md]:
```suggestion
**Are you unsure about which repository to post the bug report into?**
```

File: .github/ISSUE_TEMPLATE/1-BUG_REPORT.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+---
+
+## Bug Report
+
+<!--- ⚠️ Before you start reporting the bug... ---
+
+**Is your bug report directly related to a specific command?**
+
+If yes, make sure you are in the correct repository that contains the command you are referring to, and only create the issue in that repository.
```
Comment (danielbachhuber) [inline .github/ISSUE_TEMPLATE/1-BUG_REPORT.md]:
```suggestion
If yes, make sure you create the issue in the correct repository.
```

File: .github/ISSUE_TEMPLATE/1-BUG_REPORT.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+
+Here's a quick overview of where to find the different commands:
```
Comment (danielbachhuber) [inline .github/ISSUE_TEMPLATE/1-BUG_REPORT.md]:
```suggestion
Here's a quick overview to where each command lives:
```

File: .github/ISSUE_TEMPLATE/1-BUG_REPORT.md [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+* `wp rewrite` https://github.com/wp-cli/rewrite-command
+* `wp (role|cap) *` https://github.com/wp-cli/role-command
+* `wp scaffold *` https://github.com/wp-cli/scaffold-command
+* `wp search-replace` https://github.com/wp-cli/search-replace-command
+* `wp server` https://github.com/wp-cli/server-command
+* `wp shell` https://github.com/wp-cli/shell-command
+* `wp super-admin *` https://github.com/wp-cli/super-admin-command
+* `wp (widget|sidebar) *` https://github.com/wp-cli/widget-command
+
+If you are not in the correct repository right now, you can just close this issue/window without submitting and click through to the correct one.
```
Comment (danielbachhuber) [inline .github/ISSUE_TEMPLATE/1-BUG_REPORT.md]:
```suggestion
If you are not in the correct repository, you can just close this issue/window without submitting and navigate to the correct one.
```

File: .github/ISSUE_TEMPLATE/1-BUG_REPORT.md [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+
+**Are you unsure about what repository to post the bug report into?**
+
+Just head over to the [`wp-cli/wp-cli`](https://github.com/wp-cli/wp-cli) repository and [create a new issue in that repository](https://github.com/wp-cli/wp-cli/issues/new). The maintainers can still move the bug report into the correct repository later on.
```
Comment (danielbachhuber) [inline .github/ISSUE_TEMPLATE/1-BUG_REPORT.md]:
```suggestion
Navigate to the [`wp-cli/wp-cli`](https://github.com/wp-cli/wp-cli) repository and [create a new issue in that repository](https://github.com/wp-cli/wp-cli/issues/new). The maintainers can still move the bug report to the correct repository later on.
```

File: .github/ISSUE_TEMPLATE/1-BUG_REPORT.md [hunk 7]
```diff
@@ -0,0 +1,93 @@
...
+
+--- ✅ If you are in the correct location now... --->
+
+- [ ] Yes, I reviewed the [contribution guidelines](https://make.wordpress.org/cli/handbook/contributing/).
```
Comment (danielbachhuber) [inline .github/ISSUE_TEMPLATE/1-BUG_REPORT.md]:
```suggestion
- [ ] Yes, I reviewed and agree with the [contribution guidelines](https://make.wordpress.org/cli/handbook/contributing/).
```

File: .github/ISSUE_TEMPLATE/1-BUG_REPORT.md [hunk 8]
```diff
@@ -0,0 +1,93 @@
...
+- [ ] Yes, more specifically, I reviewed the guidelines on [how to write clear bg reports](https://make.wordpress.org/cli/handbook/bug-reports/).
+
+**Describe the current, buggy behavior**
+
+A clear and concise description of the behavior that produces a wrong result or error.
```
Comment (danielbachhuber) [inline .github/ISSUE_TEMPLATE/1-BUG_REPORT.md]:
```suggestion
A clear and concise description of the behavior that produces an incorrect result or error.
```

File: .github/ISSUE_TEMPLATE/1-BUG_REPORT.md [hunk 9]
```diff
@@ -0,0 +1,93 @@
...
+
+Remember to provide some context to make sure we're all on the same page when reasoning about this.
```
Comment (danielbachhuber) [inline .github/ISSUE_TEMPLATE/1-BUG_REPORT.md]:
```suggestion
Remember: more information is better. Please provide context on how you're running the command to make sure we're all on the same page when reasoning about this.
```

File: .github/ISSUE_TEMPLATE/1-BUG_REPORT.md [hunk 10]
```diff
@@ -0,0 +1,93 @@
...
+
+- a list of
+- steps to replicate
+- the error condition
+
+```js
+// You can also use code snippets if needed.
+```
+
+**Describe what you would expect as the correct outcome**
```
Comment (danielbachhuber) [inline .github/ISSUE_TEMPLATE/1-BUG_REPORT.md]:
```suggestion
**Describe what you expect as the correct outcome**
```

File: .github/ISSUE_TEMPLATE/1-BUG_REPORT.md [hunk 11]
```diff
@@ -0,0 +1,93 @@
...
+(Paste the output of "wp cli info" into this box)
+```
+
+**Provide a possible solution**
+
+If you happen to have a suggestion on how to fix this bug, please tell us in here.
+
+Just leave this section out if you don't know how to fix it.
+
+**Provide additional context/Screenshots**
```
Comment (danielbachhuber) [inline .github/ISSUE_TEMPLATE/1-BUG_REPORT.md]:
```suggestion
**Provide additional context/screenshots**
```

File: .github/ISSUE_TEMPLATE/4-REGULAR_RELEASE_CHECKLIST.md [hunk 12]
```diff
@@ -0,0 +1,166 @@
...
+title: ''
+labels: 'i: scope:distribution'
+assignees: 'schlessera'
+
+---
+# Regular Release Checklist - v2.x.x
+
+### Preparation
+
+- [ ] Write release post on the [Make.org CLI blog](https://make.wordpress.org/cli/wp-admin/post-new.php)
```
Comment (danielbachhuber) [inline .github/ISSUE_TEMPLATE/4-REGULAR_RELEASE_CHECKLIST.md]:
```suggestion
- [ ] Draft release post on the [make.wordpress.org CLI blog](https://make.wordpress.org/cli/wp-admin/post-new.php)
```

**Comment by schlessera** [comment]
Hmm, I should have customized the commit messages methinks...

**Pull request merged**